spec/javascripts/spec/track_submit.spec.js

```diff
@@ -0,0 +1,85 @@
+describe('A form submit tracker', function() {
+  "use strict";
+
+  var root = window,
+    tracker,
+    element;
+
+  beforeEach(function() {
+    tracker = new GOVUKAdmin.Modules.TrackSubmit();
+  });
+
+  describe('with defaults', function() {
+    beforeEach(function() {
+      element = $('\
+        <div \
+          data-track-action="form-submitted"\
+          data-track-label= "edit-link">\
+          <form class="js-track" method="post">\
```

Comment:
I've removed the `js-track` class from the form. However, regarding removing the div entirely, I think this is the standard way that `track-submit` is being implemented across GDS so I've left it in. Here are other examples of this was done: https://github.com/search?p=1&q=org%3Aalphagov+track-submit&type=Code